### DISCUSSION TITLE
Authenticate from MAUI app  using custom server built around AUTH0

### DISCUSSION THREAD

**Discussion by Holyraptor** [dotnet/maui#9904] [Q&A]
Hello. I have some troubles coming up with a solution with this issue:

I want to create a MAUI Blazor app which will call certain endpoints from my custom server. The endpoints can be accessed only by authorized user so, in order to do that, I need to authenticate. My custom web server already has an integration with AUTH0 and I can login in browser, but how do I get ahold of the authToken in the MAUI app from a blazor component.

TL;DR: My MAUI client needs an auth token from my custom server that uses Auth0.

Maybe I am missing something. I tried to find some other answers but most of them are not what I am looking for.
Thank you.
